databricks_cli/configure/cli.py

```diff
@@ -31,19 +32,39 @@
 from databricks_cli.configure.config import profile_option, get_profile_from_context, debug_option
 
 PROMPT_HOST = 'Databricks Host (should begin with https://)'
+PROMPT_RESOURCE_ID = 'Resource/Workspace ID'
 PROMPT_USERNAME = 'Username'
 PROMPT_PASSWORD = 'Password' #  NOQA
 PROMPT_TOKEN = 'Token' #  NOQA
+ENV_AAD_TOKEN = 'DATABRICKS_TOKEN'
```

Comment:
This seems like a bug?

Reply:
What's the bug here?

Reply:
The name of the env var is `DATABRICKS_TOKEN` that's a bit confusing if we're asking for the AAD token right?

Reply:
You're right. I thought about this and for the users, to make it as less confusing as possible, I think the var should be DATABRICKS_AAD_TOKEN even though we deal with it in the same as a DATABRICKS TOKEN in the backend. I've changed the variables names to make it consistent with it as well.